omnigibson/scenes/scene_base.py

```diff
@@ -249,6 +245,13 @@ def clear_systems(self):
             print(f"Clearing system: {system.name}")
             system.clear()
 
+    def clear(self):
```

Comment:
Where is this function called? `Simulator`?

Reply:
yes!